CMakeLists.txt

```
@@ -68,7 +68,7 @@ find_package(FairCMakeModules 1.0.0 REQUIRED QUIET)
 include(FairFindPackage2) # for find_package2
 
 if(BUILD_GRPC_CLIENT OR BUILD_GRPC_SERVER)
-  find_package2(PRIVATE Protobuf VERSION 3.15 REQUIRED)
+  find_package2(PRIVATE Protobuf VERSION REQUIRED CONFIG)
```

This breaks the CI, because the protobuf installed there (via system package) doesn't come with the config files.
What about something like this:
```suggestion
  find_package2(PRIVATE Protobuf CONFIG QUIET)
  if(NOT Protobuf_FOUND)
    find_package2(PRIVATE Protobuf VERSION REQUIRED)
  endif()
```
?

---

Pls make sure to read https://fairrootgroup.github.io/FairCMakeModules/latest/module/FairFindPackage2.html#find-package2. The find_package2 command with a VERSION argument and no version value is not specified!

---

See https://github.com/FairRootGroup/FairCMakeModules/blob/4690a81fa2ed13eee820e01d14ecea757ca99722/src/modules/FairFindPackage2.cmake.in#L210 - `QUIET` is always used by find_package2, so it is redundant.